Tests/Unit/Microsoft365DSC/Microsoft365DSC.EXOTransportConfig.Tests.ps1

```diff
@@ -0,0 +1,226 @@
+[CmdletBinding()]
+param(
+)
+$M365DSCTestFolder = Join-Path -Path $PSScriptRoot `
+    -ChildPath "..\..\Unit" `
+    -Resolve
+$CmdletModule = (Join-Path -Path $M365DSCTestFolder `
+        -ChildPath "\Stubs\Microsoft365.psm1" `
+        -Resolve)
+$GenericStubPath = (Join-Path -Path $M365DSCTestFolder `
+        -ChildPath "\Stubs\Generic.psm1" `
+        -Resolve)
+Import-Module -Name (Join-Path -Path $M365DSCTestFolder `
+        -ChildPath "\UnitTestHelper.psm1" `
+        -Resolve)
+
+$Global:DscHelper = New-M365DscUnitTestHelper -StubModule $CmdletModule `
+    -DscResource "EXOTransportConfig" -GenericStubModule $GenericStubPath
+Describe -Name $Global:DscHelper.DescribeHeader -Fixture {
+    InModuleScope -ModuleName $Global:DscHelper.ModuleName -ScriptBlock {
+        Invoke-Command -ScriptBlock $Global:DscHelper.InitializeScript -NoNewScope
+
+        BeforeAll {
+            $secpasswd = ConvertTo-SecureString "test@password1" -AsPlainText -Force
+            $Credential = New-Object System.Management.Automation.PSCredential ("tenantadmin", $secpasswd)
+
+            Mock -CommandName Update-M365DSCExportAuthenticationResults -MockWith {
+                return @{}
+            }
+
+            Mock -CommandName Get-M365DSCExportContentForResource -MockWith {
+
+            }
+
+            Mock -CommandName Confirm-M365DSCDependencies -MockWith {
+
+            }
+
+            Mock -CommandName New-M365DSCConnection -MockWith {
+                return "Credential"
+            }
+
+            Mock -CommandName Get-PSSession -MockWith {
+
+            }
+
+            Mock -CommandName Remove-PSSession -MockWith {
+
+            }
+
+            Mock -CommandName Set-TransportConfig -MockWith {
+
+            }
+        }
+
+        # Test contexts
+        Context -Name "Values are already in the desired state" -Fixture {
```

Comment:
You should also add tests for scenarios where the code goes into a catch block and also test the scenario when no parameters are specified (see other comment)

Reply:
Added test for catch block

Reply:
Did you commit these tests? I cannot see the new tests anywhere

Reply:
I had to remove the test for catch block again since it did not work as expected. Did not found a working solution yet, but the tests are structured in the same way as other resources (e.g. EXOOrganizationConfig). 
Do you have an example for a resource with tests implemented in the way you expect?